subsys/bluetooth/audio/pacs.c

```diff
@@ -257,6 +259,20 @@ static int set_available_contexts(uint16_t contexts, uint16_t *available,
 	return 0;
 }
 
+static int set_supported_contexts(uint16_t contexts, uint16_t *supported)
+{
+	int err;
+
+	*supported = contexts;
+
+	err = k_work_schedule(&supported_contexts_work, PAC_NOTIFY_TIMEOUT);
```

Comment:
Should it have been reschedule instead? 

```suggestion
	err = k_work_reschedule(&supported_contexts_work, PAC_NOTIFY_TIMEOUT);
```

Reply:
Done